_data/identity.yml

```diff
@@ -113,6 +113,16 @@ websites:
       software: Yes
       doc: http://www.onelogin.com/product/strong-authentication/two-factor-authentication/
 
+    - name: PassHub
+      url: https://www.passhub.net/
+      img: PassHub.png
+      tfa: Yes
+      software: Yes
+      hardware: Yes
+      exceptions:
+          text: "hardware tokens only work on desktop."
```

Comment:
If hardware only works on desktops, what happens if someone tries to login via another device?
Also capitalize "Hardware"

Reply:
As of today, you can not connect hardware token to a mobile device, so if your account was created with hardware token, it will be accessible only on a PC or Mac with WWPass Security Pack installed, and either NFC reader or USB port. If your account created with phone-based token (WWPass Passkey app from Appstore or Playstore), it can be accessed on PC, Mac, phones, tablets and even coffee machines and refrigerators, as long as they have compatible web browsers, capable of performing client side encryption computations.